inc/class-sitemaps.php

```diff
@@ -57,9 +58,9 @@ public function register_sitemaps() {
 		/**
 		 * Filters the list of registered sitemap providers.
 		 *
+		 * @param array $providers Array of Core_Sitemap_Provider objects.
```

Comment:
Your editor is going to keep doing this isn't it 😆

Reply:
No this is historic, promise :D